### ISSUE TITLE
vortex doesn't start

### ISSUE THREAD

**Issue by codeman101** [lutris/lutris#2133]
I've been struggling for hours trying to get Vortex mod manager running on wine directly with no luck. I finally discovered that a Lutris install script was created for it. Two in fact but neither of them work. After installing the later one of the two this is what get as terminal output when I run Lutris from the terminal. 

> Running /home/frank/.local/share/lutris/runners/wine/tkg-4.0-x86_64/bin/wine /home/frank/Games/vortex-mod-manager/drive_c/Program Files/Black Tree Gaming Ltd/Vortex/Vortex.exe
> 
> info: steam not found error=File not found. 'Software\Valve\Steam'
> Waiting on children
> All children gone
> Exit with returncode 0
> 2019-05-13 23:39:14,747: Game still running (state: running)
> 2019-05-13 23:39:14,748: Stopping Vortex Mod Manager (wine)
> 

Before even realizing that there's now a lutris script I even made an issue on the wine bugzilla about this program not starting and even upgraded my wine version from 4.6 to 4.8.1 via building it from source.

I feel stressed out from trying to solve this for so long (despite the fact that I don't mean non-stop) Any ideas?

I'm more than willing to give any terminal output needed just tell me what to do to get it.

**Comment by codeman101** [comment]
Someone the wine bugzilla helped me solve the issue of installing Vortex via wine directly. I had to have winetricks install allfonts. Maybe this should be added to the Lutris scripts.

**Comment by tannisroot** [comment]
Perhaps.

**Comment by codeman101** [comment]
@tannisroot 

I don't blame you for closing my issue. I almost did it myself with that second post I made. The only reason I didn't was because I thought the issue could be directed towards adding that to the script but even if it was the mod manager seems impractical to use under wine which is why I don't blame you for closing it. :)

**Comment by tannisroot** [comment]
Just to clarify, I think fonts should be installed by default too.

**Comment by codeman101** [comment]
@tannisroot 

In that case is the reason you closed the issue because you think I should make a new issue about having that added to the script? I suppose I could but to be honest I don't feel it's worth it yet at least not until it becomes possible to add Vortex to the same proton install as the skyrim game. (which is currently an open issue on the proton github) For now it's much easier to add the mods by pasting the files directly into the Data folder and enabling them from the in game mod menu in skyrim special edition.
